django_template_i18n_lint.py

```diff
@@ -162,7 +162,7 @@ def replace_strings(filename, overwrite=False, force=False, accept=[]):
                 full_text_lines.append(leading_whitespace)
 
                 # Find location of first letter
-                lineno, charpos = location(template, offset+m.span()[0])
+                lineno, charpos = location(string, offset+m.span()[0])
```

Comment:
The `template` variable isn't in the scope of the function, and replacing it with `string` as it's called in the for loop fixes the issue.